### PR TITLE
Release notes 5.1.4

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.1.md [hunk 1]
```diff
@@ -11,6 +11,71 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+## Release 5.1.4
+**Release Date:** June 23, 2021
+
+### New Features
+
+- **I95-35414 Refresh actions now available for individual sections on the Router Page:** The Device interface, Network Interface, and Peer Paths table sections now can be refreshed independently.
+------
+- **I95-38244 The Routers Page is easier to Search:** Added a column selector and a search matching system to make the search function more granular. 
+------
+- **I95-38445 GUI Session Capture:** Added pages to the user interface that allow you to view and configure capture information.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:23]: consider providing a link to https://docs.128technology.com/docs/ts_packet_capture/#selective-packet-capture

File: docs/release_notes_128t_5.1.md [hunk 2]
```diff
@@ -11,6 +11,71 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-40532 Added Tenant prefix support on network interface:** Provides a simpler way to configure the tenant prefixes on a per branch basis.
+
+### Resolved Issues
+
+- **I95-39713 Access policy object screen has incorrect heading text:** The heading for an Access Policy setting has been changed to display the correct heading text.
+------
+- **I95-39954 Conductor Service Generation Service Policy cannot be set:** Resolved a management-service-generation validation error. 
+------
+- **I95-40144 EoSVR with outbound only sessions showing errors:** Resolved an issue where EoSVR with outbound only sessions may show errors when fpm is turned on, and will not contribute to fpm data.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:37]: FPM should be capitalized.

File: docs/release_notes_128t_5.1.md [hunk 3]
```diff
@@ -11,6 +11,71 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-40239 CVE-2021-26937:** This vulnerability has been resolved.
+------
+- **I95-40241 RHSA-2020:1180:** Fixed RHSA-2020:1180 by deprecating package ImageMagick.
+------
+- **I95-40242 CESA-2021:0856 advisory:** This vulnerability has been resolved. 
+------
+- **I95-40304 Allow for duplicate domain names:** When same domain name appears for multiple categories the system will preserve the first such entry and ignore the rest.
+------
+- **I95-40334 `show service-path` shows bgposvr for an EoSVR interface:** Resolved the issue where ethernet-over-svr routes would be displayed as bgp-over-svr in the output.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:49]: should be BGPoSVR

File: docs/release_notes_128t_5.1.md [hunk 4]
```diff
@@ -11,6 +11,71 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-40242 CESA-2021:0856 advisory:** This vulnerability has been resolved. 
+------
+- **I95-40304 Allow for duplicate domain names:** When same domain name appears for multiple categories the system will preserve the first such entry and ignore the rest.
+------
+- **I95-40334 `show service-path` shows bgposvr for an EoSVR interface:** Resolved the issue where ethernet-over-svr routes would be displayed as bgp-over-svr in the output. 
+------
+- **I95-40349 Improve Session Not Found Message:** The message is now more user-friendly. 
+------
+- **I95-40380 RoutersPage facet selector doesn't work without a search value present:** Removed a condition causing the facets to be ignored when the search bar is empty.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:53]: Should be "Routers Page".
I don't believe we use contractions

File: docs/release_notes_128t_5.1.md [hunk 5]
```diff
@@ -11,6 +11,71 @@ If you do not see an issue listed below, it may have been resolved in another re
...
+------
+- **I95-40473 API Username not being recorded:** Resolved an issue where the "modify_user" event was omitting the "fullName" modified field.
+------
+- **I9540489 ISO missing 128T-minion-connector rpm:** The 128T-minion-connector plugin rpm was not included in the 5.1 OTP ISO. This has been corrected in the 128T-5.1.3-1.el7.OTP.v3.x86_64.iso ISO.
+------
+- **I95-40477 Import certificate webserver doesn't copy the private key:** This issue has been resolved.
+------
+- **I95-40669 Reverse SSH config file only generated if remote login enabled:** The reverse SSH config file is now generated correctly. 
+------
+- **I95-40682 the GUI login page keeps focus on whichever element was last selected:** Login page will now focus username field after invalid login.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:71]: The should be capitalized

Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

File: docs/release_notes_128t_5.1.md (hunk 4 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: fixed

File: docs/release_notes_128t_5.1.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: fixed

File: docs/release_notes_128t_5.1.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

File: docs/release_notes_128t_5.1.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

File: docs/about_releases.md [hunk 6]
```diff
@@ -13,6 +13,7 @@ However, issues resolved in `4.3.12`, which was released on 3/12/2021 are not ad
 
 | Version                                          | Release Date       | GA Date            | End of Maintenance | End of Support     |
 | ------------------------------------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
+| [5.1.4](release_notes_128t_5.1.md#release-513)   | June 23, 2021      |                    | Not Applicable*    | May 24, 2022       |
```
Comment (MichaelBaj) [inline docs/about_releases.md:16]: The date will need to be updated

File: docs/plugin_intro.md [hunk 7]
```diff
@@ -40,6 +40,15 @@ Installed plugin can be removed from the UI by using the `Uninstall` button on t
...
 ![128T Plugin Uninstall](/img/plugin_uninstall.gif)
 
+#### Running Plugin Commands
+Commands that are bundled with installed plugins may be run from the GUI using the Plugin Commands interface. Selecting Plugins from the Administration menu opens the Plugin State tab, which shows the state of any currently installed plugins. 
+
+![Plugins State Tab](/img/plugin_state_tab.png)
+
+Use the Commands tab to execute commands available for the individual plugins. 
+
+![Plugin Commands Tab](/img/plugin_commands_tab.png)
```
Comment (MichaelBaj) [inline docs/plugin_intro.md:50]: The Mist Agent plugin is not currently distributed and is currently under development. We should showcase some of our standard plugin offerings. @agrawalkaushik should be able to provide a relevant image

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/about_releases.md (hunk 6 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/about_releases.md]: Date has been updated per today's staff meeting.

File: docs/plugin_intro.md (hunk 7 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/plugin_intro.md]: messaged with Kaushik, and have removed the graphic.

**Review by MichaelBaj — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/plugin_intro.md [hunk 8]
```diff
@@ -40,6 +40,13 @@ Installed plugin can be removed from the UI by using the `Uninstall` button on t
 
 ![128T Plugin Uninstall](/img/plugin_uninstall.gif)
 
+#### Running Plugin Commands
+Commands that are bundled with installed plugins may be run from the GUI using the Plugin Commands interface. Selecting Plugins from the Administration menu opens the Plugin State tab, which shows the state of any currently installed plugins. 
+
+![Plugins State Tab](/img/plugin_state_tab.png)
```
Comment (agrawalkaushik) [inline docs/plugin_intro.md:46]: @Chr1st0ph3rTurn3r This is also showing the state for the MIST plugin. The state is supported for a few other plugins - I would recommend using the `dns-app-id` plugin for this instead. I dont have any setup I can downgrade but maybe someone on field_eng can provide a snapshot for you

**Review by agrawalkaushik — CHANGES_REQUESTED** [review]
remove the mist-agent state output

**Review by MichaelBaj — DISMISSED** [review]

File: docs/plugin_intro.md (hunk 8 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/plugin_intro.md]: done

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart — DISMISSED** [review]

**Review by agrawalkaushik — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**